noxfile.py

```diff
@@ -14,12 +14,12 @@
 
 
 def run_common(session: Session, coverage_threshold: float) -> None:
-    if session.python == "3.12":
-        session.install("-e .[dev,extra,dask,modin]")
-    elif session.python != "3.8":
-        session.install("-e .[dev,extra,dask,modin,pyspark,ibis]")
```

Comment:
I changed the order of `if-elif` because it seems a bit more readable to me

Since it is a personal take I can revert if you want :)